### PR TITLE
Nightwatch webserver setup + tear down

### PR DESCRIPTION
**Pull request by dawehner**
* [x] Starts the webserver automatically
* [x] Tears down Drupal
* [x] Test finish

### CODE REVIEW

File: core/tests/Drupal/Nightwatch/globals.js [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+      process.env.BASE_URL = 'http://localhost:8888';
+      phpWebServer = spawn(commandAsWebserver('php'), ['-S', 'localhost:8888', '.ht.router.php'], {cwd: '../'});
+    }
     done();
   },
   after: (done) => {
     if (!JSON.parse(process.env.CHROME_STANDALONE)) {
       chromedriver.stop();
     }
+    if (phpWebServer) {
```
Comment (alexpott) [inline core/tests/Drupal/Nightwatch/globals.js]: I'm not sure if this is the right way to do this - I've done too much PHP :)

File: core/tests/Drupal/Nightwatch/Tests/index.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
 module.exports = {
   'Test page': (browser) => {
-    let installedDbPrefix = null;
     browser
       .installDrupal('\\Drupal\\TestSite\\TestSiteInstallTestScript', (dbPrefix) => {
```
Comment (alexpott) [inline core/tests/Drupal/Nightwatch/Tests/index.js]: It kinda bothers me that installing and tearing down is part of the test. I think we should be doing this in beforeEach and afterEach. But then of course we need to be able to run a script against the test site. Which means we need to go back to the command and break up install into two steps. One to install the command and one to run the script.

**Comment by alexpott** [comment]
https://github.com/jsdrupal/nightwatch/pull/9/commits/1459bfa531606af16f279ebfa4a95e1b28f193bb fixes done-ness and also re-organises the test into something that make more sense to me.

File: core/tests/Drupal/Nightwatch/globals.js (hunk 1 above)
Comment (dawehner) [reply core/tests/Drupal/Nightwatch/globals.js]: That's totally alright.

**Pull request merged**